CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 * Cover `src/components/Loading/Loading.js` file by RTL/Jest tests. Refs UIREQ-885.
 * Leverage cookie-based authentication in all API requests. Refs UIREQ-861.
 * Update `circulation` okapi interface to `14.0` version. Refs UIREQ-954.
+* Added requestDate token. Refs CIRC-1784.
```

Comment:
Could you please update CHANGELOG.md how i understood we should change from  `* Added requestDate token. Refs CIRC-1784.` to `* Added requestDate token. Refs UIREQ-962.`